app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -102,4 +102,14 @@ def vm_create_snapshot(vm, options = {})
       service.process_task(response.body)
     end
   end
+
+  def vm_remove_snapshot(vm, options = {})
+    vm_remove_all_snapshots(vm, options)
```

Comment:
```
vm_remove_all_snapshots(vm)
```

No need to pass options forward because it's not used anywhere in vm_remove_all_snapshots function